packages/apktool.vm/apktool.vm.nuspec

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="utf-8"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2015/06/nuspec.xsd">
+  <metadata>
+    <id>apktool.vm</id>
+    <version>2.6.0.20220225</version>
```

Comment:
Date shouldn't be included here as the forth segment is not used in the dependency. From the [documentation](https://github.com/mandiant/VM-Packages/wiki/Contributing#metapackages):
> When a metapackage is named after and installs/configures a specific tool (for instance cygwin.vm that installs/configures cygwin) lock the dependency version (e.g., version="[3.2.0]") and use the locked version as the metapackage's version. If the tool's version uses the 4th segment (e.g., version="[2.0.1.20210213]") update the metapackage's version to use the current date in the 4th segment (i.e., <version>2.0.1.20220224</version>).

Or is the documentation not clear enough? 🤔

Reply:
If we're going to wind up using the fourth segment anyway, perhaps we should update the documentation to say to always use the fourth segment as the current date? @Ana06